maestro-ios-driver/src/main/kotlin/util/LocalSimulatorUtils.kt

```diff
@@ -411,23 +413,38 @@ object LocalSimulatorUtils {
 
     fun startScreenRecording(deviceId: String): ScreenRecording {
         val tempDir = createTempDirectory()
-        val script = LocalSimulatorUtils::class.java.getResource("/screenrecord.sh")!!.file
-        val recording = File(tempDir.toFile(), "screenrecording.mov")
+        val inputStream = LocalSimulatorUtils::class.java.getResourceAsStream("/screenrecord.sh")
+        if (inputStream != null) {
+            // Create a temporary file
+            val tempFile = File.createTempFile("screenrecord", ".sh")
```

Comment:
As long as the script is smaller than 260kB, this would be simpler and faster:

```
ProcessBuilder(listOf("bash", "-c", String(inputStream.readAll))
```

The 260kB limit is the maximum size a process arguments can be on MacOS (with default os settings. Which is plenty, the script now is 1kB.